applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -1354,6 +1387,105 @@ int handle_at_cellpos(enum at_cmd_type cmd_type)
 	return err;
 }
 
+/**@brief handle AT#XWIFIPOS commands
+ *  AT#XWIFIPOS=<op>[,<ssid>,<mac>[,<ssid>,<mac>[...]]]
+ *  AT#XWIFIPOS?
+ *  AT#XWIFIPOS=?
+ */
+int handle_at_wifipos(enum at_cmd_type cmd_type)
+{
+	int err = -EINVAL;
+	uint32_t count = at_params_valid_count_get(&at_param_list);
+	char mac_addr_str[WIFI_MAC_ADDR_STR_LEN];
+	uint32_t mac_addr[WIFI_MAC_ADDR_LEN];
+	struct wifi_scan_result *ap;
+	uint16_t op;
+	int index = 1;
+	int size;
+
+	switch (cmd_type) {
+	case AT_CMD_TYPE_SET_COMMAND:
+		err = at_params_unsigned_short_get(&at_param_list, index, &op);
+		if (err < 0) {
+			return err;
+		}
+
+		if (op == WIFIPOS_START && nrf_cloud_ready && run_type == RUN_TYPE_NONE) {
+			if (count < (2 + NRF_CLOUD_LOCATION_WIFI_AP_CNT_MIN * 2)) {
```

Comment:
a comment here describing why this math needs to be done would be helpful

Reply:
Currently SLM only send SSID and MAC address of each AP to nRF Cloud, which makes the NRF_CLOUD_LOCATION_WIFI_AP_CNT_MIN * 2. The first 2 is index 0 for AT command and index 1 for op code.

Will rephrase